src/server/utilities/cspHeader/index.js

```diff
@@ -348,6 +348,20 @@ const directives = {
       'https://static.bbci.co.uk', // STY includes
     ],
   },
+  mediaSrc: {
+    ampLive: [
+      'https://news.files.bbci.co.uk', // VJ includes may load .mp3 files
+    ],
+    canonicalLive: [
+      'https://news.files.bbci.co.uk', // VJ includes may load .mp3 files
+    ],
+    ampNonLive: [
+      'https://news.files.bbci.co.uk', // VJ includes may load .mp3 files
```

Comment:
Should the non-live be `https://news.test.files.bbci.co.uk` @eagerterrier ?

Reply:
Presumably we need the live one on test too/regardless, as we can render live content on our test env?

Reply:
I think I agree with Jim. If we have that ability it would be really handy to preview live on test